client/state/purchases/selectors.js

```diff
@@ -5,3 +5,6 @@ import find from 'lodash/find';
 export const getByPurchaseId = ( state, id ) => (
 	assign( {}, state, { data: find( state.data, { id } ) } )
 );
+
+export const isFetchingUserPurchases = state => state.purchases.isFetchingUserPurchases;
```

Comment:
We could add super simple unit tests for those method just to keep in sync with how other state selectors are covered with tests.